context.go

```diff
@@ -15,7 +15,6 @@ type Context struct {
 	serviceID  ServiceID
 	manager    *serviceManager
 	bucketName string
-	network.Dispatcher
```

Comment:
I'm not sure all parts of cothority still work with this change! Can you verify that, please?

Reply:
Cothority builds and passes "make test" with this change in place.